src/formatter.ts

```diff
@@ -67,13 +74,21 @@ function createContexts<T>(
   makeContext: makeContextFn<T>,
   addToContext: addToContextFn<T>
 ): AlignmentContexts<T> {
-  const resolutionMultiplier = Formatter.getResolutionMultiplier(voices);
+  let resolutionMultiplier = 0;
 
   // Initialize tick maps.
   const tickToContextMap: Record<number, T> = {};
   const tickList: number[] = [];
   const contexts: T[] = [];
 
+  if (voices.length == 0)
+    return {
+      map: tickToContextMap,
+      array: contexts,
+      list: tickList.sort((a, b) => a - b),
+      resolutionMultiplier,
+    };
+  resolutionMultiplier = Formatter.getResolutionMultiplier(voices);
```

Comment:
Maybe restructure here to keep `resolutionMultiplier` a const.

```typescript
  // let resolutionMultiplier = 0; <-- REMOVE THIS

  const tickToContextMap: Record<number, T> = {};
  const tickList: number[] = [];
  const contexts: T[] = [];

  if (voices.length == 0)
    return {
      map: tickToContextMap,
      array: contexts,
      list: tickList.sort((a, b) => a - b),
      resolutionMultiplier: 0,
    };
  const resolutionMultiplier = Formatter.getResolutionMultiplier(voices);
```

Reply:
done